index.html

```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <section class="card_section">
+        <div class="cards_container">
+            <div class="card_img">
+                <img src="./images/SnapshootPortfolio.svg" alt="card image">
+            </div>
+            <div class="card_text">
+                <h1>Tonic</h1>
+                <div class="card_text_flex">
+                    <p><strong>CANOPY</strong></p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>Band End Dev</p>
+                    <img src="images/Vector.png" alt="vector">
+                    <p>2015</p>
+                </div>
+                <p class="card_paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="card_lang_flex">
+                    <p><strong>html</strong></p>
+                    <p><strong>css</strong></p>
+                    <p><strong>javascript</strong></p>
+                </div>
```

Comment:
Please ensure that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`